co2eq/purchase/footprints.yml

```diff
@@ -30,6 +30,37 @@ _children:
         unit: EUR
   Electronics:
     _children:
+      'Average smartphone':
+        icon: mobile
+        intensityKilograms: 32
+        source: 'https://www.ademe.fr/sites/default/files/assets/documents/poids_carbone-biens-equipement-201809-rapport.pdf'
+        unit: item
+        usage: 20%
+        _children:
+          'Feature phone':
```

Comment:
It probably shouldn't be a child of smartphone?

Reply:
Yup - let's call it Phone (generic)